packages/web-components/fast-foundation/src/data-grid/data-grid-cell.template.ts

```diff
@@ -14,6 +14,14 @@ export function dataGridCellTemplate<T extends FASTDataGridCell>(): ElementViewT
         <template
             tabindex="-1"
             role="${x => DataGridCellRole[x.cellType] ?? DataGridCellRole.default}"
+            :classList="
```

Comment:
This change seems unrelated to the PR, but we've been moving away from classes on the template level. If there are styles that need this somewhere can they be written to use the attribute selector instead?

Reply:
I think ViewTemplate changed how classes are bound so binding to the the actual property may be necessary here. I'll try and dig up the PR where this was done.

Reply:
Found it. I misremembered some of the details so this can be the class attribute or the property depending on the behavior you want. https://github.com/microsoft/fast/issues/6413

Reply:
The change I was thinking of is #6425

Reply:
Weren't using the classes in the example anyway, removed and fixed issue with prop selector.